components/AppFooter/AppFooter.tsx

```diff
@@ -0,0 +1,164 @@
+import { createStyles, Text, Container, ActionIcon, Group } from '@mantine/core';
+import AppLink from '../AppLink/AppLink';
+import Image from "next/image";
+import LogoSmall from "../../public/logo_small.png";
+import { DiscordIcon } from "../AppHeader/DiscordIcon";
+import { SteamIcon } from "./SteamIcon";
+import { FooterTextLinks } from "./FooterTextLinks"
```

Comment:
To be more consistent with other modules, I would expect camelCase for `footerTextLinks`